ansible/inventory.yml

```diff
@@ -144,7 +148,23 @@ hosts:
 
     - macstadium:
         macos11.0-arm64-1: {ip: 199.7.163.9, user: administrator}
```

Comment:
Does this host also need a `remote_env `set?

Reply:
possibly? but this is a DTK (like `-2`) which I dont expect us to keep for much longer so it may never actually be used

Reply:
Was just querying the inconsistency.